pysparkling/context.py

```diff
@@ -608,3 +611,29 @@ def __call__(self, data):
             length = struct.unpack(self.length_fmt, prefix)[0]
             package, data = data[:length], data[length:]
             yield package
+
+
+class SparkContext:
+    """Accepts the same initialization parameters as pyspark, but redirects everything to Context."""
```

Comment:
Why not subclassing Context instead?

Reply:
Because I actually wanted to make SparkContext a singleton. And as far as I can see, Context is not a singleton.
And I went a far way out to try to copy the pyspark API into SparkContext, but finally ended up with the current implementation.

Long story short: inheriting & checking for single use in __new__ should be better :-). I updated the code as such.